rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -164,14 +223,28 @@ def mock_requests_get(url: str, timeout: int):  # pylint: disable=unused-argumen
     assert 'shutter' not in data[TEST_ADDR2]
 
     # Test cache files are created
-    for protocol_name in AIRDROPS:
+    for protocol_name in ('uniswap', '1inch', 'grain', 'shapeshift', 'cow_gnosis', 'diva', 'shutter'):  # noqa: E501
         assert (database.user_data_dir.parent / 'airdrops' / f'{protocol_name}.csv').is_file()
 
 
-@pytest.mark.parametrize('airdrop_list', [NOT_CSV_WEBPAGE])
-def test_airdrop_fail(mock_airdrop_list, database):  # pylint: disable=unused-argument
-    with pytest.raises(UnableToDecryptRemoteData):
+def test_airdrop_fail(database):
+    with (
+        patch('rotkehlchen.chain.ethereum.airdrops.requests.get', side_effect=_mock_airdrop_list),
+        pytest.raises(UnableToDecryptRemoteData),
+    ):
         check_airdrops(
             addresses=[TEST_ADDR1],
             database=database,
         )
+
+
+def test_valid_airdrop_index_schema():
+    airdrops_data, poap_airdrops_data = fetch_airdrops_metadata()
+    for protocol_name, airdrop_data in airdrops_data.items():
+        assert isinstance(protocol_name, str)
+        assert airdrop_data.csv_url.endswith('.csv')
+    for protocol_name, airdrop_data in poap_airdrops_data.items():
+        assert isinstance(protocol_name, str)
+        for i in range(3):
+            assert isinstance(airdrop_data[i], str)
+        assert airdrop_data[0].endswith('.json')
```

Comment:
this is slow and I'm not sure that is worth to run on every test. I was thinking more in adding CI to the data repo and have this test running there every time we add a new airdrop